include/libultraship/libultra/gbi.h

```diff
@@ -1,41 +1,6 @@
-#include "mbi.h"
-
-#ifndef ULTRA64_GBI_H
-#define ULTRA64_GBI_H
-
-#ifdef _MSC_VER
-#ifndef u8
-#define u8 uint8_t
-#endif
-
-#ifndef u16
-#define u16 uint16_t
-#endif
+#pragma once
 
-#ifndef u32
-#define u32 uint32_t
-#endif
-
-#ifndef u64
-#define u64 uint64_t
-#endif
-
-#ifndef s8
-#define s8 int8_t
-#endif
-
-#ifndef s16
-#define s16 int16_t
-#endif
-
-#ifndef s32
-#define s32 int32_t
-#endif
-
-#ifndef s64
-#define s64 int64_t
-#endif
-#endif
```

Comment:
was this intentional? if so it's not mentioned in the PR description and this goes beyond header guard stuff

Reply:
They were pointless since the other headers defined these types already. I can revert if though.

Reply:
2 thoughts:
* something somewhere in some port might be relying on those defines being there
* it's not part of standardizing on pragma, so if we want to remove those it should happen in a separate PR